brianlong/www.validators.app
# 179311434 validator identity and withdrawer

#### What's this PR do?
Daemon for gathering vote_accounts details (authorized withdrawer and validator identity) and give a warn sign for the validator if those two values match

#### How should this be manually tested?
run the daemon /daemons/gather_vote_account_details_daemon.rb and see if it updates vote_account and score

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/179311434

#### Task completed checklist:
- [Y] Is there appropriate test coverage?
- [Y] Did I take into consideration possible security vulnerabilities?
- [Y] Are the controllers secure?
- [Y] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
